gstatus/glusterlib/display_status.py

```diff
@@ -111,4 +114,18 @@ def _build_status(data):
     return(cluster+vols)
 
 
+def print_json(data):
+    import json
+    from datetime import datetime
 
+    # Build the cluster data for json
+    gstatus = {}
+    gstatus['cluster_status']  = data.cluster_status
+    gstatus['glfs_version']    = data.glusterfs_version
+    gstatus['node_count']      = data.nodes
+    gstatus['nodes_active']    = data.nodes_reachable
+    gstatus['volume_count']    = data.volume_count
+    gstatus['volumes_started'] = data.volumes_started
+    if data.volume_count:
+        gstatus['volume_summary'] = data.volume_data
+    print("%s %s"%(datetime.now(), json.dumps(gstatus)))
```

Comment:
I think it would be good to have entire output as json.

```
json.dumps({
    "last_updated": datetime.now(),
    "data": gstatus
})
```

Reply:
It was printed separately to be backward compatible. For example:
@victorarthae splits the data at: 
https://github.com/MrCirca/zabbix-glusterfs/blob/master/gstatus_discovery.py#L8

Reply:
What do you think?

Reply:
The advantage of having full output as JSON is the option to use with other CLI tools. For example,

```
# JSON Pretty print
gstatus -a -ojson | python -m json.tool
```

or using with `jq` tool.

```
gstatus -a -o json | jq <options>
```

Reply:
Link to `jq` tool https://stedolan.github.io/jq/

Reply:
Ack. Looks good.